src/services/DfusionService.ts

```diff
@@ -4,6 +4,8 @@ import { Logger, ContractEventLog, tokenList, Erc20Contract, BatchExchangeContra
 
 import packageJson from '../../package.json'
 import { BigNumber } from 'bignumber.js'
+import { version as jsVersion } from '@gnosis.pm/dex-js/package.json'
```

Comment:
nit: would name `dexJsVersion`

Reply:
Yes, when I wrote that I thought it was a bad name. I'll take the suggestion